libs/sdk-backend-tiger/src/convertors/fromBackend/AnalyticalDashboardConverter.ts

```diff
@@ -2,20 +2,22 @@
 import { IListedDashboard } from "@gooddata/sdk-backend-spi";
 
 import {
-    AnalyticalDashboardResourceSchema,
-    AnalyticalDashboardResourcesResponseSchema,
+    AnalyticalDashboardAttributes,
+    AnalyticalDashboards,
+    AnalyticalDashboardsItem,
 } from "@gooddata/api-client-tiger";
+
 import { idRef } from "@gooddata/sdk-model";
 
 export const convertAnalyticalDashboard = (
-    analyticalDashboard: AnalyticalDashboardResourceSchema,
+    analyticalDashboard: AnalyticalDashboardsItem,
 ): IListedDashboard => {
-    const { id, attributes, links } = analyticalDashboard;
+    const attributes = analyticalDashboard.attributes as AnalyticalDashboardAttributes;
     const { title, description } = attributes;
     return {
-        ref: idRef(id, "analyticalDashboard"),
-        uri: (links as any).self,
-        identifier: id,
+        ref: idRef(analyticalDashboard.id, "analyticalDashboard"),
+        uri: (analyticalDashboard.links as any).self,
```

Comment:
Do we have some better type than `any` for this? There are several places with this `links as any` stuff, so if you know a better type, please update it elsewhere too :)

Reply:
we have type `Links` -> adapted everywhere